lib/model/model.go

```diff
@@ -59,6 +59,11 @@ type service interface {
 	getState() (folderState, time.Time, error)
 }
 
+type AvailabilityInfo struct {
```

Comment:
Could be just `type Availability`? (I forget why protocol.FileInfo and protocol.BlockInfo aren't called just protocol.File and protocol.Block but I *think* that's an unfortunate accident from like v0.2 where it was all in one package and a Block was something else already...)